src/Language/Haskell/LSP/VFS.hs

```diff
@@ -17,6 +17,9 @@ module Language.Haskell.LSP.VFS
   (
     VFS(..)
```

Comment:
Should we hide the fields of the VFS record?

Reply:
My opinion is no, it would be very annoying for someone if they had to add a missing function themselves.